keyboards/und/pcm-t-k0-windows.xml

```diff
@@ -0,0 +1,183 @@
+		<map iso="B08" to="."/>
+		<map iso="B09" to=";"/>
+		<map iso="B10" to="'"/>
+	</keyMap>
+	<keyMap modifiers="shift">
+		<map iso="E00" to="̀"/>
+		<map iso="E01" to="!"/>
+		<map iso="E02" to="@"/>
+		<map iso="E03" to="#"/>
+		<map iso="E04" to="$"/>
+		<map iso="E05" to="₦"/>
+		<map iso="E06" to="%"/>
+		<map iso="E07" to="&amp;"/>
+		<map iso="E08" to="*"/>
+		<map iso="E09" to="("/>
+		<map iso="E10" to=")"/>
+		<map iso="E11" to="_"/>
+		<map iso="E12" to="+"/>
+		<map iso="D01" to="́"/>
+		<map iso="D02" to="W"/>
+		<map iso="D03" to="E"/>
+		<map iso="D04" to="R"/>
+		<map iso="D05" to="T"/>
+		<map iso="D06" to="Y"/>
+		<map iso="D07" to="U"/>
+		<map iso="D08" to="I"/>
+		<map iso="D09" to="O"/>
+		<map iso="D10" to="P"/>
+		<map iso="D11" to="{"/>
+		<map iso="D12" to="}"/>
+		<map iso="C01" to="A"/>
+		<map iso="C02" to="S"/>
+		<map iso="C03" to="D"/>
+		<map iso="C04" to="F"/>
+		<map iso="C05" to="G"/>
+		<map iso="C06" to="H"/>
+		<map iso="C07" to="J"/>
+		<map iso="C08" to="K"/>
+		<map iso="C09" to="L"/>
+		<map iso="C10" to="Ọ"/>
+		<map iso="C11" to="Ẹ"/>
+		<map iso="C12" to="?"/>
+		<map iso="B00" to="?"/>
+		<map iso="B01" to="Z"/>
+		<map iso="B02" to="C"/>
+		<map iso="B03" to="V"/>
+		<map iso="B04" to="B"/>
+		<map iso="B05" to="N"/>
+		<map iso="B06" to="M"/>
+		<map iso="B07" to="&lt;"/>
+		<map iso="B08" to="&gt;"/>
+		<map iso="B09" to=":"/>
+		<map iso="B10" to="&quot;"/>
+	</keyMap>
+	<keyMap modifiers="caps">
+                <map iso="E00" to="`"/>
+		<map iso="E01" to="1"/>
+		<map iso="E02" to="2"/>
+		<map iso="E03" to="3"/>
+		<map iso="E04" to="4"/>
+		<map iso="E05" to="5"/>
+		<map iso="E06" to="6"/>
+		<map iso="E07" to="7"/>
+		<map iso="E08" to="8"/>
+		<map iso="E09" to="9"/>
+		<map iso="E10" to="0"/>
+		<map iso="E11" to="-"/>
+		<map iso="E12" to="="/>
+		<map iso="D01" to="Q"/>
+		<map iso="D02" to="W"/>
+		<map iso="D03" to="E"/>
+		<map iso="D04" to="R"/>
+		<map iso="D05" to="T"/>
+		<map iso="D06" to="Y"/>
+		<map iso="D07" to="U"/>
+		<map iso="D08" to="I"/>
+		<map iso="D09" to="O"/>
+		<map iso="D10" to="P"/>
+		<map iso="D11" to="["/>
+		<map iso="D12" to="]"/>
+		<map iso="C01" to="A"/>
+		<map iso="C02" to="S"/>
+		<map iso="C03" to="D"/>
+		<map iso="C04" to="F"/>
+		<map iso="C05" to="G"/>
+		<map iso="C06" to="H"/>
+		<map iso="C07" to="J"/>
+		<map iso="C08" to="K"/>
+		<map iso="C09" to="L"/>
+		<map iso="C10" to="Ọ"/>
+		<map iso="C11" to="Ẹ"/>
+		<map iso="C12" to="/"/>
+		<map iso="B00" to="/"/>
+		<map iso="B01" to="Z"/>
+		<map iso="B02" to="C"/>
+		<map iso="B03" to="V"/>
+		<map iso="B04" to="B"/>
+		<map iso="B05" to="N"/>
+		<map iso="B06" to="M"/>
+		<map iso="B07" to=","/>
+		<map iso="B08" to="."/>
+		<map iso="B09" to=";"/>
+		<map iso="B10" to="'"/>
+	</keyMap>
+	<transforms type="simple">
+		<!-- Change vowels with appended acute, grave, and dot below combiners to combined forms -->
+		<transform from="a\u{300}" to="à"/>
+		<transform from="E\u{300}" to="È"/>
```

Comment:
Would be nice to have this set of transforms ordered alphabetically and then by code point of diacritic.

Reply:
This PR was supposed to be just a merge, but that's OK-

@xadxura how do we want to handle this? @sven-oly  Do you want to make a new PR into the keyboards' branch?

Reply:
@xadxura if this is a stylistic thing, it's probably something that should be handled by automated tools. I don't know if CLDRModify already touches these ( @macchiati may know) but that would be the way to do it, so that they're automatically kept in some order.

Reply:
> Would be nice to have this set of transforms ordered alphabetically and then by code point of diacritic.

filed CLDR-14193 to track this in tools.